ansible/roles/prometheus/vars/main.yml

```diff
@@ -150,3 +150,9 @@ blackbox_jobs:
   - name: icmp
     module: icmp
     targets: "{{ dom0_hosts | list }}"
+
+prometheus_aws_access_key: "{{ lookup('amazon.aws.aws_ssm', '/oonidevops/secrets/ooni_monitoring/access_key', profile='oonidevops_user_dev') }}"
+prometheus_aws_secret_key: "{{ lookup('amazon.aws.aws_ssm', '/oonidevops/secrets/ooni_monitoring/secret_key', profile='oonidevops_user_dev') }}"
```

Comment:
Probably these variables need to be called `prometheus_aws_secret_key_dev` and `_prod` since we have to look them up using different profiles?

Reply:
Yes, you are right, we need a prod/dev version for these variables!
Thanks for noticing